### PR TITLE
personcard to tsx

### PR DESCRIPTION
**Pull request by jwinnfeild2517**
## What does this PR do

Convert person card to typeScript

### CODE REVIEW

File: src/components/Cards/PersonCard/index.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import styled, { css } from 'styled-components';
 
 import PersonHeadShot from '../shared/PersonHeadShot/PersonHeadShot';
 import { color, font, fontSize, lineHeight, mixins, spacing, withThemes } from '../../../styles';
 
+type ModeType = {
+  mode: string
+}
```
Comment (ttornese) [inline src/components/Cards/PersonCard/index.tsx:9]: we only have dark and light mode, so a better type here might be something like: `type ModeType = 'dark' | 'light'`. this is a good candidate for a global type as well

File: src/components/Cards/PersonCard/index.tsx [hunk 2]
```diff
@@ -70,11 +73,25 @@ const PersonCardDescriptionTheme = {
...
+const PersonCardDescription = styled.div<ModeType>`
   ${withThemes(PersonCardDescriptionTheme)}
 `;
 
-const PersonCard = ({ description, imgAlt, imgCloudinaryId, mode, name }) => (
+type PersonCardProps = {
+  description: string,
+  imgAlt?: string,
+  imgCloudinaryId: string,
+  mode?: string,
```
Comment (ttornese) [inline src/components/Cards/PersonCard/index.tsx:84]: if you redefine `ModeType` to what i made in my comment above you could then set the type of this `mode` prop to `ModeType`

File: src/components/Cards/PersonCard/index.tsx (hunk 1 above)
Comment (jwinnfeild2517) [reply src/components/Cards/PersonCard/index.tsx]: 👍

File: src/components/Cards/PersonCard/index.tsx (hunk 2 above)
Comment (jwinnfeild2517) [reply src/components/Cards/PersonCard/index.tsx]: 👍

**Review by ttornese — APPROVED** [review]

**Pull request merged**